lib/server/generate.js

```diff
@@ -443,21 +425,25 @@ function execute() {
             str
           );
         }
+
+        // write to base level
```

Comment:
I am thinking there is something funny going on in this section.

Reply:
But I am not 100% sure yet.

Reply:
This was a change I made in order to generate default pages from English pages. Original code was to copy English pages to one level above. However those pages has {language} property as 'en' which causes links wrong. So I made the change to generate instead. Couldn't think of a problem it would cause, but let me run some tests first.